ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1608,3 +1608,137 @@
                 default-excludes: true
                 fingerprint: false
                 only-if-success: false
+      - 'cloud-{name}-{test_name}-aws':
```

Comment:
We can use `cloud-{name}-{test_name}-cleanup` for all related templates.

Reply:
The template uses the `CLUSTERNAME` to delete a cluster, which is not working for deleting the EC2 in AWS.
```
- job-template:
     name: 'cloud-{name}-{test_name}-cleanup'
     node: '{node}'
     triggers: '{triggers}'
     builders: '{builders}'
     parameters:
     - string:
         default: ''
         description: The cluster name of the last finished build.
         name: CLUSTERNAME
         trim: 'false'
```